local_intelligence_hub/settings.py

```diff
@@ -237,3 +243,12 @@
         "debug_toolbar.panels.logging.LoggingPanel",
         "debug_toolbar.panels.redirects.RedirectsPanel",
     ]
+
+# CK Section
+
+one_hour = timedelta(hours=1)
+GQL_AUTH = GqlAuthSettings(
+    JWT_EXPIRATION_DELTA=one_hour,
```

Comment:
Wonder if we make this longer for prototype users, else it might get annoying while we are in test mode?

Perhaps a week instead?